settings.php

```diff
@@ -58,6 +58,24 @@
  */
 $settings['update_free_access'] = FALSE;
 
+/**
+ * Fall back to HTTP for Update Manager.
+ *
+ * If your Backdrop site fails to connect to updates.backdropcms.org using HTTPS
+ * to fetch the update status of Backdrop core and/or contrib projects, you may
+ * uncomment this setting (and make sure it is set to "TRUE"), to allow an
+ * insecure fallback to HTTP.
+ *
+ * SECURITY WARNING: Enabling this setting will open your site up to potential
+ * man-in-the-middle attacks! You should instead attempt to resolve the issues,
+ * and only use this option as a last resort.
+ *
+ * @see https://docs.backdropcms.org/documentation/system-requirements#php
+ * @see https://en.wikipedia.org/wiki/Man-in-the-middle_attack
+ * @see _update_build_fetch_url()
```

Comment:
The last `@see` ... are people really supposed to search the code base for a helper function? :thinking: 

References to functions are great withing the code base (for developers), but this is settings.php (for site builders).

Reply:
There are multiple other places within settings.php where we use `@see` to point to functions (although not helper functions indeed). Happy to remove this if others also find it problematic 👍🏼